content/jobs/2023/technical-content-specialist.md

```diff
@@ -0,0 +1,60 @@
++++
+authors =  ["James Munroe"]
+title = "Technical Content Specialist"
+work_description = [
+"Create documentation on cloud-based interactive computing.",
+"Curate documentation and resources from open source and open science projects."
+]
+salary_range = "$75,000 - $85,000"
+location = "Preferred within the UTC-7 to UTC+3 time zones"
+url_apply = "https://forms.gle/H43CsVUoF7e2C2Nm8"
+deadline = "We will begin reviewing applications around **August 2, 2023**, and will accept them on a rolling basis until the position is filled."
+date = "2023-07-12"  # This is the date we _post_ this job.
+open = "true"  # SET TO FALSE IF THE JOB IS NO LONGER OPEN
++++
+
+2i2c manages, supports, and builds community-centric infrastructure for interactive computing in the cloud with partner communities in research and education.
+
+We’re looking for a Technical Content Specialist** **that will create and curate documentation that supports interactive computing and cloud based open science. It intersects job titles such as “technical writer” and “content creator” while emphasizing previous experience working with open source science and related tools and a desire to share that knowledge.
+
+&lt;!-- Insert metadata for this job -->
+&lt;!-- defined in layouts/shortcodes/job_details.html -->
+{{% job_details %}}
+
+If you’re interested in advancing your career at an impact-driven non-profit that is dedicated to open communities and public knowledge, then read on…
+
+## About you…
+Below are several skills that will make somebody well-suited for this role. **You should apply even if you do not have all of these skills**. We expect any new hires to learn and grow into this role over time. If you aren’t sure whether you have the right skills for this job, you should just apply!
+
+### Necessary qualities
+* Demonstrated excellence in communicating complex technical information to learners with a different levels of background and experience
+* Demonstrated experience in using open-science tools (e.g. Python or R) in a scientific or scholarly domain
+* Willingness to continually learn and share emerging technologies and techniques in the interactive computing and open science ecosystems
+
+### Useful qualities
+* Experience with open source workflows, research and educational contexts, and an understanding of the value that cloud-based infrastructure provides
+* Experience with the Jupyter ecosystem and other tools for interactive computing
+* Experience working with distributed service teams that use asynchronous methods of communication
+* Experience collaborating and coordinating work via online platforms and distributed revision control
+* Ability to communicate in Spanish, Portuguese, or French
```

Comment:
communicate = verbally and written?
Just curious...

Reply:
Both have value!